packages/jellyfish-transaction/src/script/defi/dftx_token.ts

```diff
@@ -22,8 +21,20 @@ export class CTokenMint extends ComposableBuffer<TokenMint> {
 
   composers (tm: TokenMint): BufferComposer[] {
     return [
-      ComposableBuffer.varUInt(() => tm.tokenId, v => tm.tokenId = v),
-      ComposableBuffer.satoshiAsBigNumber(() => tm.amount, v => tm.amount = v)
+      ComposableBuffer.varUIntArray(() => tm.balances, v => tm.balances = v, v => new CTokenBalance(v))
+    ]
+  }
+}
+
+/**
+ * Composable TokenBalance, C stands for Composable.
+ * Immutable by design, bi-directional fromBuffer, toBuffer deep composer.
+ */
+export class CTokenBalance extends ComposableBuffer<TokenBalance> {
+  composers (tb: TokenBalance): BufferComposer[] {
+    return [
+      ComposableBuffer.uInt32(() => tb.token, v => tb.token = v),
+      ComposableBuffer.satoshiAsBigNumber(() => tb.amount, v => tb.amount = v)
     ]
   }
 }
```

Comment:
same for this:
https://github.com/DeFiCh/jellyfish/blob/3fb5abc3f8db0e272e9940d22224ff747d0bd728/packages/jellyfish-transaction/src/script/defi/dftx_balance.ts#L18-L25

Reply:
ohh noted!!